terraform/modules/nitro/main.tf

```diff
@@ -0,0 +1,38 @@
+provider "aws" {
```

Comment:
This folder adds a module for creating nitro based EC2 instances using the same configuration that we have been using previously.  It does, however, allow the user to "bring their own" AWS keys.